### PR TITLE
aws: Add elb_account_id data source

### PR DESCRIPTION
**Pull request by radeksimko**
I'm not sure if data source is the right place for such static lookup, but I can't think of a better place right now.

http://docs.aws.amazon.com/elasticloadbalancing/latest/classic/enable-access-logs.html#attach-bucket-policy

### CODE REVIEW

**Comment by jen20** [comment]
I am generally :+1: on this - exposing as much information as possible via data sources to prevent hard coded values is a good idea IMO!

**Comment by jen20** [comment]
LGTM pending Travis and running acctests - sorry about the merge conflict there!

**Comment by jen20** [comment]
```
› envchain aws.terraform make testacc TEST=./builtin/providers/aws TESTARGS="-run TestAccAWSElbAccountId_basic"
==> Checking that code complies with gofmt requirements...
/Users/James/Code/go/bin/stringer
go generate $(go list ./... | grep -v /terraform/vendor/)
2016/08/16 11:43:39 Generated command/internal_plugin_list.go
TF_ACC=1 go test ./builtin/providers/aws -v -run TestAccAWSElbAccountId_basic -timeout 120m
=== RUN   TestAccAWSElbAccountId_basic
--- PASS: TestAccAWSElbAccountId_basic (19.17s)
PASS
ok     	github.com/hashicorp/terraform/builtin/providers/aws   	19.187s
```

**Comment by apparentlymart** [comment]
I was initially kinda confused by the name of this, thinking to myself "What is an ELB account?"

After reading the linked docs I understand now that this is the account id of the ELB service itself, for each region.

So with that definition in mind, could we instead call this `aws_elb_service_account`? (The `_id` suffix is also redundant, since id is an attribute so it causes "stutter".)

So this would look like:

```hcl
data "aws_elb_service_account" "current_region" {
}
data "aws_iam_policy_document" "elb_access" {
  statement {
    principals {
        type        = "AWS"
        identifiers = "${data.aws_elb_service_account.current_region.id}"
    }
    # etc
  }
}
```

(Sorry this is a bit of a nit... just concerned that the naming as merged will be misleading and cause people to misunderstand what this is for.)

File: website/source/docs/providers/aws/d/elb_account_id.html.markdown [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    instance_protocol = "http"
+    lb_port = 80
+    lb_protocol = "http"
+  }
+}
+```
+
+## Argument Reference
+
+* `region` - (Optional) Region of a given AWS ELB Account
```
Comment (apparentlymart) [inline website/source/docs/providers/aws/d/elb_account_id.html.markdown]: Maybe clearer to say: "Name of the region whose ELB account id is desired. If not specified, default's to the region from the AWS provider configuration."

File: website/source/docs/providers/aws/d/elb_account_id.html.markdown [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+
+## Attributes Reference
+
+* `id` - Account ID
```
Comment (apparentlymart) [inline website/source/docs/providers/aws/d/elb_account_id.html.markdown]: Maybe clearer to say: "The id of the ELB service account in the selected region."

**Comment by radeksimko** [comment]
Thanks for all the comments, agreed to all points. I'll address that in a separate PR.

**Comment by radeksimko** [comment]
^ @stack72 FYI - I know there's a decent portion of code you copied from here, so it would be good to make similar changes in https://github.com/hashicorp/terraform/pull/8224

**Pull request merged**